dist/style.css

```diff
@@ -51,34 +51,37 @@ body {
   text-align: center;
 }
 
+#logo-img {
+  background-image: url('SeekADesign_VectorLogo.svg');
```

Comment:
**For relative paths:**

`./` means "starting at the current directory"
`../` means "go backwards one directory"
`../../` means "go backwards two directories"

Please put this file back in the `./dist/assets/logos` directory, and change the path here to:

`./dist/assets/logos/SeekADesign_VectorLogo.svg`